Gemfile

```diff
@@ -18,9 +18,7 @@ group :acceptance_testing do
   gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 3.0')
 end
 
-if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-  gem "scooter", *location_for(ENV['SCOOTER_VERSION'] || '~> 3.0')
-end
+gem "scooter", *location_for(ENV['SCOOTER_VERSION'] || '~> 3.0')
```

Comment:
Scooter was only publicly available as of 4.1.0:

https://rubygems.org/gems/scooter